search-service/src/test/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeServiceTests.kt

```diff
@@ -248,7 +249,7 @@ class TemporalEntityAttributeServiceTests : WithTimescaleContainer, WithKafkaCon
         temporalEntityAttributeService.createEntityTemporalReferences(rawEntity, listOf(APIC_COMPOUND_CONTEXT))
             .shouldSucceed()
 
-        val createdAt = ZonedDateTime.now()
+        val createdAt = ngsiLdDateTime()
```

Comment:
you are modifying it here... but not in the tests you are adding (see for instance line 300)...